backend/shared/src/helpers/generate-and-update-avatar-urls.ts

```diff
@@ -1,27 +1,8 @@
-import * as admin from 'firebase-admin'
-
-import { getStorage, Storage } from 'firebase-admin/storage'
-import { User } from 'common/user'
+import { Storage } from 'firebase-admin/storage'
 import { DOMAIN } from 'common/envs/constants'
 
 type Bucket = ReturnType<InstanceType<typeof Storage>['bucket']>
 
-const firestore = admin.firestore()
-export const generateAndUpdateAvatarUrls = async (users: User[]) => {
-  const storage = getStorage()
-  const bucket = storage.bucket()
-  console.log('bucket name', bucket.name)
-
-  await Promise.all(
-    users.map(async (user) => {
-      const userDoc = firestore.collection('users').doc(user.id)
-      console.log('backfilling user avatar:', user.id)
-      const avatarUrl = await generateAvatarUrl(user.id, user.name, bucket)
```

Comment:
Why remove this? It's nice to have an auto-generated avatar url. And remove an optional field

Reply:
this particular code isn't used anywhere. maybe was part of a script?
anyways it reads from the users collection and I don't really feel like migrating or testing it